hphp/runtime/ext/extension-registry.cpp

```diff
@@ -292,6 +292,17 @@ void scanExtensions(IMarker& mark) {
   }
 }
 
+
+std::vector<std::pair<std::string, std::set<std::string>>>
+getAllExtensionSystemlibSources() {
+  std::vector<std::pair<std::string, std::set<std::string>>> ret;
+  for (auto& ext : s_ordered) {
```

Comment:
Note to self when updating this for ext_closure: This needs to be checking that the extensions are actually enabled before adding them to the list.